src/cart/crt_iv.c

```diff
@@ -2738,7 +2738,7 @@ bulk_update_transfer_done_aux(const struct crt_bulk_cb_info *info)
 					&cb_info->buc_iv_value,
 					cb_info->buc_user_priv);
 		output->rc = rc;
-		rc = crt_reply_send(info->bci_bulk_desc->bd_rpc);
+		crt_reply_send(info->bci_bulk_desc->bd_rpc);
```

Comment:
This isn't right, crt_reply_send can return an error so it should be checked/handled.

Reply:
I take this back, we almost never check the rc from crt_reply_send()  This change at least makes the source consistent, but we should probably look at how to correctly handle this class of errors.